vmdb/app/models/container_group.rb

```diff
@@ -9,7 +9,7 @@ class ContainerGroup < ActiveRecord::Base
   has_many :containers, :dependent => :destroy
   has_many :container_definitions, :dependent => :destroy
   belongs_to  :ext_management_system, :foreign_key => "ems_id"
-  has_many :labels, :class_name => CustomAttribute, :as => :resource, :conditions => {:section => "labels"}
+  has_many :labels, -> { where(:section => "labels") }, :class_name => CustomAttribute, :as => :resource
```

Comment:
Ugh it's so weird looking :-\ (but oh well)  would almost prefer it it were a regular block

```ruby
has_many(:spam_comments, class_name: 'Comment') { where spam: true }
```